src/Transition/CreateNewIDFUsingRulesV9_3_0.f90

```diff
@@ -1845,4 +1875,139 @@ SUBROUTINE SortUnique(StrArray, Size, Order)
     END DO
   END DO
 
-END SUBROUTINE SortUnique
\ No newline at end of file
+END SUBROUTINE SortUnique
+
+SUBROUTINE UpdateEMSFunctionName(InOutArg, OldName, NewName, NoDiffArg)
```

Comment:
Wow - this got overly complicated.

Reply:
👍 Getting paid by the line over here! 

I'm happy to make changes, though, but I might need some suggestions from a Fortran expert.

Reply:
If it ain't broke . . . I'll test it shortly.

Reply:
@mitchute I do not have Fortran compiler in my machine, so I cannot make complete review. I kindly request @mjwitte to review. Thanks.